ices/80409.rs

```diff
@@ -0,0 +1,33 @@
+use std::marker::PhantomData;
```

Comment:
There are some hard tabs, please run rustfmt for this.

Reply:
Fixed in 126fd7f